apps/Gruntfile.js

```diff
@@ -513,6 +513,8 @@ describe('entry tests', () => {
     'pd/professional_learning_landing/index': './src/sites/studio/pages/pd/professional_learning_landing/index.js',
     'pd/regional_partner_contact/new': './src/sites/studio/pages/pd/regional_partner_contact/new.js',
 
+    'pd/international_optin/new': './src/sites/studio/pages/pd/international_optin/new.js',
```

Comment:
nit: s/optin/opt_in

And elsewhere. optin isn't a word, nor a common abbreviation AFAIK, so I'd go with `opt_in` and `OptIn` over `optin` and `Optin`